source/localizable/routing/defining-your-routes.md

```diff
@@ -183,6 +183,26 @@ If the user navigates to `/post/5`, the route will then have the `post_id` of
 `5` to use to load the correct post. In the next section, [Specifying a Route's
 Model](../specifying-a-routes-model), you will learn more about how to load a model.
 
+### Serializing data into dynamic segments
+
+If you want to control how data gets serialized into your route's dynamic segments,
+you can define a `serialize` function.
+
+```app/router.js
+function serializePost(model) {
+  return { post_id: model.id };
+}
+
+Router.map(function() {
+  this.route('post', {
+    path: '/post/:post_id',
+    serialize: serializePost
+  });
+});
+```
+
+If no `serialize` function is defined, then a straight property lookup on the model will be used for each segment instead.
```

Comment:
Can you give an example? Or maybe inverse it, first explain how the default works, and then explain how you can customize. I believe you can check the `model()` hook documentation for a similar flow.